public/pages/DetectorDetail/containers/DetectorDetail.tsx

```diff
@@ -0,0 +1,340 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React, { useState, useEffect, useCallback } from 'react';
+import {
+  EuiTabs,
+  EuiTab,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiHealth,
+  EuiOverlayMask,
+} from '@elastic/eui';
+// @ts-ignore
+import { toastNotifications } from 'ui/notify';
+import { get } from 'lodash';
+import { RouteComponentProps, Switch, Route, Redirect } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { useFetchDetectorInfo } from '../../createDetector/hooks/useFetchDetectorInfo';
+import { useHideSideNavBar } from '../../main/hooks/useHideSideNavBar';
+import {
+  deleteDetector,
+  startDetector,
+  stopDetector,
+} from '../../../redux/reducers/ad';
+import { getErrorMessage, Listener } from '../../../utils/utils';
+//@ts-ignore
+import chrome from 'ui/chrome';
+import { darkModeEnabled } from '../../../utils/kibanaUtils';
+import { BREADCRUMBS } from '../../../utils/constants';
+import { DetectorControls } from '../components/DetectorControls';
+import moment from 'moment';
+import { ConfirmModal } from '../components/ConfirmModal/ConfirmModal';
+import { useFetchMonitorInfo } from '../hooks/useFetchMonitorInfo';
+import { MonitorCallout } from '../components/MonitorCallout/MonitorCallout';
+
+export interface DetectorRouterProps {
+  detectorId?: string;
+}
+interface DetectorDetailProps
+  extends RouteComponentProps<DetectorRouterProps> {}
+
+const tabs = [
+  {
+    id: 'results',
+    name: 'Anomaly Results',
+    route: 'results',
+  },
+  {
+    id: 'configurations',
+    name: 'Detector configuration',
+    route: 'configurations',
+  },
+];
```

Comment:
Make it as a constant. and make `id` as constant as well.

Reply:
Thanks, will change